submodules/rapidxml/rapidxml_print.hpp

```diff
@@ -192,6 +192,7 @@ namespace rapidxml
         template<class OutIt, class Ch>
         inline OutIt print_attributes(OutIt out, const xml_node<Ch> *node, int flags)
         {
+            (void)flags;

```

Comment:
Is this to silence an unused warning, but you want to keep the parameter name? (rather than leaving it unamed or just removing the parameter.)

Reply:
yeah, thanks, fixed by removing the argument name